Framework/Kernel/src/NexusHDF5Descriptor.cpp

```
@@ -105,7 +106,9 @@ herr_t readStringAttributeN(hid_t attr, char *data, int maxlen) {
   char *vdat = NULL;
   iRet = readStringAttribute(attr, &vdat);
   if (iRet >= 0) {
+    GNU_DIAG_OFF("stringop-truncation")
```

Are we hiding an actual error here?

---

Will fix that later.